config/routes.rb

```diff
@@ -2,6 +2,12 @@
   devise_for :users
 
   namespace :editorial do
+    get '/news' => 'news#index', as: :news_articles
```

Comment:
I have had to do the latter as we already specify the `@form` object. However specifying the URL explicitly has worked.